bricksrc/threshold.py

```diff
@@ -0,0 +1,1275 @@
+                                    BRICK.Discharge_Air_Temperature_Low_Reset_Threshold,
+                                ],
+                                BRICK.hasQuantity: BRICK.Temperature,
+                                BRICK.hasSubstance: BRICK.Supply_Air,
+                                "parents": [
+                                    BRICK.Temperature_Low_Reset_Threshold,
+                                ],
+                                "tags": [
+                                    TAG.Point,
+                                    TAG.Supply,
+                                    TAG.Discharge,
+                                    TAG.Air,
+                                    TAG.Temperature,
+                                    TAG.Differential,
+                                    TAG.Reset,
+                                    TAG.Threshold,
+                                    TAG.Low,
+                                ],
+                            },
+                        },
+                    },
+                },
+            },
+            "Supply_Air_Flow_Reset_Threshold": {
+                "aliases": [
+                    BRICK.Discharge_Air_Flow_Reset_Threshold,
+                ],
+                BRICK.hasQuantity: QUDTQK.VolumeFlowRate,
+                BRICK.hasSubstance: BRICK.Discharge_Air,
+                "tags": [
+                    TAG.Point,
+                    TAG.Supply,
+                    TAG.Discharge,
+                    TAG.Air,
+                    TAG.Flow,
+                    TAG.Reset,
+                    TAG.Threshold,
+                ],
+                "subclasses": {
+                    "Supply_Air_Flow_High_Reset_Threshold": {
+                        "aliases": [
+                            BRICK.Discharge_Air_Flow_High_Reset_Threshold,
+                        ],
+                        "tags": [
+                            TAG.Point,
+                            TAG.Supply,
+                            TAG.Discharge,
+                            TAG.Air,
+                            TAG.Flow,
+                            TAG.Reset,
+                            TAG.Threshold,
+                            TAG.High,
+                        ],
+                    },
+                    "Supply_Air_Flow_Low_Reset_Threshold": {
+                        "aliases": [
+                            BRICK.Discharge_Air_Flow_Low_Reset_Threshold,
+                        ],
+                        "tags": [
+                            TAG.Point,
+                            TAG.Supply,
+                            TAG.Discharge,
+                            TAG.Air,
+                            TAG.Flow,
+                            TAG.Reset,
+                            TAG.Threshold,
+                            TAG.Low,
+                        ],
+                    },
+                },
+            },
+            "Upper_Threshold_Threshold": {
```

Comment:
Is Threshold_Threshold a typo?

Reply:
yes this was a regex search and replace that I missed